tests/ctst/world/Zenko.ts

```diff
@@ -94,7 +99,7 @@ export interface ZenkoWorldParameters {
 export default class Zenko extends World<ZenkoWorldParameters> {
     private readonly command: string = '';
 
-    private result: object = {};
+    private result: ApiResult = {} as ApiResult;
```

Comment:
Not sure the `as` is needed here if you already create the variable of the right type, it should be properly ineferred by typescript